src/AppInstallerCommonCore/DateTime.cpp

```diff
@@ -49,6 +49,19 @@ namespace AppInstaller::Utility
         return result;
     }
 
+    std::string GetCurrentDateForARP()
+    {
+        auto now = std::chrono::system_clock::now();
+        std::time_t tt = std::chrono::system_clock::to_time_t(now);
+
+        struct tm newTime;
+        localtime_s(&newTime, &tt);
+
+        std::stringstream ss;
+        ss << std::put_time(&newTime, "%Y%m%d");
+        return ss.str();

```

Comment:
Do you have to use `std::chrono::system_clock::now()`? Or would this work?
```suggestion
        std::time_t now = std::time(0);
        std::stringstream ss;
        ss << std::put_time(std::localtime(&now), "%Y%m%d");
        return ss.str();
```
(Just trying to learn, so this might not be right)

Reply:
Looks correct to me (although `nullptr` is preferred over `0`).

Although if we convert to C++20, we can just get the time formatting goodness and not use the icky C stuff.